Gemfile.lock

```diff
@@ -371,3 +371,4 @@ DEPENDENCIES
 
 BUNDLED WITH
    1.10.6
+   
```

Comment:
Oh damn. I don't understand how this is creeping back in again. I actually copied what is on master this time around so I don't miss out. 😞